build/networking/README_DOCKER.md

```diff
@@ -139,50 +139,55 @@ configuration file settings and inner workings!
 If above commands are successful, at this point you should have your IPDK
 container up and running, and should see the container prompt like below:
 
-```text
-    root@c5efb1a949ad ~]#
+```bash
+  root@c5efb1a949ad ~]#
 ```
 
 `c5efb1a949ad` is your container id as displayed in command `docker ps -a`
 
-
 As your Infrap4d process should be up and running, you could see the infrap4d
 process with the `ps -ef | grep infrap4d` command.
 
 ## Section 2: Running example use case
 
-### Section 2.1: Running example setup with TAP ports.
+### Section 2.1: Running example setup with TAP ports
+
 Infrap4d process is already started with `ipdk start -d` or with manual
 steps mentioned above.
 Start example usecase by connecting to docker and running script `/root/scripts/rundemo_TAP_IO.sh`,
 this script will:
-  - Creates TAP ports and moves to two different namespaces.
-  - Create a forwarding pipeline binary.
-  - Load the target with a forwarding pipeline.
-  - Configure rules.
-  - Test traffic with Ping between TAP ports.
 
+* Creates TAP ports and moves to two different namespaces
```

Comment:
_Create_, not _Creates_

Each verb should be in first-person singular (_Create, Load, Configure, Test_), not third-person (_Creates_).

Reply:
Fixed